frontend/resources/styles/common/refactor/common-dashboard.scss

```diff
@@ -33,13 +33,6 @@
     }
   }
 
-  svg {
-    fill: $db-secondary;
-    height: $s-12;
-    margin-right: $s-4;
-    width: $s-12;
-  }
-
```

Comment:
We can do this because the only `.dashboard-header` that was using SVGs was the projects' one